wpeverest/everest-forms
# Fix - Signature field display with smart tags

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://wpeverest.helpmonks.com/mailbox/5dca366ef1d5a91590eed32a/assigned/6089e918ac17fd7f568716f2

### How to test the changes in this Pull Request:

1. In email message use a smart tag to display the form field in email like this and send email.
![image](https://user-images.githubusercontent.com/3774827/116547294-6c839400-a912-11eb-86d9-e912159db06f.png)
2. If the image is displayed then that's all this PR does a job!!

### Types of changes:




### Other information:




### Changelog entry

> Fix - Signature field display with smart tags.
